projects/kubernetes/kubelet/kubelet.sh

```diff
@@ -21,6 +21,12 @@ if [ ! -e /var/lib/cni/.opt.defaults-extracted ] ; then
     touch /var/lib/cni/.opt.defaults-extracted
 fi
 
+if [ ! -e /var/lib/cni/.opt.configs-extracted ] ; then
+    mkdir -p /var/lib/cni/etc/net.d
+    cp /var/config/cni/etc/net.d/* /var/lib/cni/etc/net.d/
```

Comment:
What happens if `/var/config/cni/etc/net.d` doesn't exist. (Hint: nothing good)

Please name the stamp `.etc-net.d-configs-extracted` (the `opt` in the example you cribbed from is because it is populating `/opt/cni`.